packages/sdk/src/build/build-get-collection.js

```diff
@@ -0,0 +1,11 @@
+import {pipe, Ok, makeGetCollection} from "../interaction/interaction.js"
+
+export function getCollection(id = null) {
+  return pipe([
+    makeGetCollection,
+    ix => {
+      ix.collection.id = id
+      return Ok(ix)
```

Comment:
We need to stop using `Ok` and `Bad` please. Instead please `return ix` instead of `return Ok(ix)` and `throw new Error(errorMsg)` instead of `return Bad(errorMsg)`

Reply:
Updated !